commands/system/help.js

```diff
@@ -0,0 +1,326 @@
+            if (level < this.client.levelCache[c.conf.permLevel]) return;
+            output = output + `\n\`${message.settings.prefix + c.help.name}\` | ${c.help.description.length > 80 ? `${c.help.description.slice(0, 80)}...` : c.help.description}`;
+          }
+          num = num + 1;
+        }
+      }
+
+      if (num) {
+        finalpage = Math.ceil(num / perpage);
+        embed.setTitle(`Page ${page}/${Math.ceil(num / perpage)}`)
+          .setDescription(`A list of commands in the ${type} category.\n(Total of ${num} commands in this category)\n\nTo get help on a specific command do \`${message.settings.prefix}help <command>\`\n\n${num > 10 && pg === 1 ? `To view more commands do\` ${message.settings.prefix}help <category> 2\`` : ""}`)
+          .addField("Commands", output);
+      }
+    }
+
+    if (this.client.commands.has(type) || this.client.commands.some(command => command.conf.aliases.includes(type))) {
+      const cm = this.client.commands.get(type) || this.client.commands.get(this.client.aliases.get(type));
+      if (level < this.client.levelCache[cm.conf.permLevel]) return;
+      embed.setTitle(cm.help.name)
+        .addField("Command description", cm.help.description)
+        .addField("Command usage", `\`${cm.help.usage}\``)
+        .addField("Command aliases", cm.conf.aliases.length === 0 ? "None" : cm.conf.aliases.join(", "));
+    }
+
+    const msg2 = await message.channel.send(embed);
+    if (!message.guild.me.hasPermission(["MANAGE_MESSAGES"])) {
+      await message.channel.send("I don\"t have permission to remove reactions, please do this manually.");
+    }
+    if (msg2.embeds[0].title) {
+      await msg2.react("⏮");
+      await msg2.react("◀");
+      await msg2.react("⏹");
+      await msg2.react("▶");
+      await msg2.react("⏭");
+      await msg2.react("🔢");
+    }
+
+    const nextpage = msg2.createReactionCollector(
+      (reaction, user) => reaction.emoji.name === "▶" && user.id === message.author.id,
+      { time: 1500000 },
+    );
+
+    const backpage = msg2.createReactionCollector(
+      (reaction, user) => reaction.emoji.name === "◀" && user.id === message.author.id,
+      { time: 1500000 },
+    );
+
+    const firstpage = msg2.createReactionCollector(
+      (reaction, user) => reaction.emoji.name === "⏮" && user.id === message.author.id,
+      { time: 1500000 },
+    );
+
+    const lastpage = msg2.createReactionCollector(
+      (reaction, user) => reaction.emoji.name === "⏭" && user.id === message.author.id,
+      { time: 1500000 },
+    );
+
+    const numberpages = msg2.createReactionCollector(
+      (reaction, user) => reaction.emoji.name === "🔢" && user.id === message.author.id,
+      { time: 1500000 },
+    );
+
+    const stop = msg2.createReactionCollector(
+      (reaction, user) => reaction.emoji.name === "⏹" && user.id === message.author.id,
+      { time: 1500000 },
+    );
+
+    nextpage.on("collect", (r) => {
```

Comment:
The following collector events are extremely hardcoded. Unify them into one. Use the array I suggested before to check if the reaction is one of the "whitelisted" ones.

The code is incredibly duplicated, and same requested changes fall into every each of them.